### PR TITLE
{bio}[system/system] USEARCH v11.0.667

### PR DESCRIPTION
**Pull request by stevecdavis**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by stevecdavis** [comment]
Just wanted to leave a warning that the software requires you to register/license it so it gets licensed to an email and when it runs it displays who it's licensed to.

File: easybuild/easyconfigs/u/USEARCH/USEARCH-11.0.667-i86linux32.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+homepage = 'https://www.drive5.com/usearch/index.html'
+description = """USEARCH is a unique sequence analysis tool which offers search and clustering algorithms that are
+ often orders of magnitude faster than BLAST."""
+
+toolchain = SYSTEM
+
+# download via https://www.drive5.com/usearch/download.html
+local_usearch_bin = 'usearch%s_%s' % (version, versionsuffix[1:])
+sources = [local_usearch_bin]
+checksums = ['9cc2864b1fb0f731386c92d61324a947d8a8afabe3b04f9939b9062c99fcb917']
```
Comment (smoors) [inline easybuild/easyconfigs/u/USEARCH/USEARCH-11.0.667-i86linux32.eb:16]: @stevecdavis iiuc the email address is included in the binary?
I filled out the university cluster license with 2 different email addresses, and each time got the link to the binary, but with a different checksum (and different from yours).
if so, the checksum should probably be removed:
```suggestion
checksums = [None]
```

File: easybuild/easyconfigs/u/USEARCH/USEARCH-11.0.667-i86linux32.eb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+install_cmd = "mkdir -p %%(installdir)s/bin && mv %s %%(installdir)s/bin/%%(namelower)s && " % local_usearch_bin
+install_cmd += "chmod a+rx %(installdir)s/bin/%(namelower)s"
```
Comment (smoors) [inline easybuild/easyconfigs/u/USEARCH/USEARCH-11.0.667-i86linux32.eb:19]: the licence page explicitly says:
>I agree to set execute-only permissions --x--x--x on the binary.

so, to be on the safe side, probably better to do this:
```suggestion
install_cmd += "chmod a=x %(installdir)s/bin/%(namelower)s"
```

File: easybuild/easyconfigs/u/USEARCH/USEARCH-11.0.667-i86linux32.eb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+modloadmsg = """USEARCH (32-bit) is licensed software but
+ can be used for free by individuals when registered. A 64-bit version is
+ also available. For more information see http://www.drive5.com/usearch .
+"""
+
+sanity_check_paths = {
+    'files': ['bin/%(namelower)s'],
```
Comment (smoors) [inline easybuild/easyconfigs/u/USEARCH/USEARCH-11.0.667-i86linux32.eb:27]: as this is a sanity check, you should use the name of the binary explicitly here, not the template.
also, if possible, can you add a short `sanity_check_commands` as well?

**Review by smoors — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/u/USEARCH/USEARCH-11.0.667-i86linux32.eb (hunk 1 above)
Comment (stevecdavis) [reply easybuild/easyconfigs/u/USEARCH/USEARCH-11.0.667-i86linux32.eb]: Made the change and resubmitted.

File: easybuild/easyconfigs/u/USEARCH/USEARCH-11.0.667-i86linux32.eb (hunk 3 above)
Comment (stevecdavis) [reply easybuild/easyconfigs/u/USEARCH/USEARCH-11.0.667-i86linux32.eb:30]: added: 
sanity_check_commands = ["usearch --version"]
and resubmitted

File: easybuild/easyconfigs/u/USEARCH/USEARCH-11.0.667-i86linux32.eb (hunk 2 above)
Comment (stevecdavis) [reply easybuild/easyconfigs/u/USEARCH/USEARCH-11.0.667-i86linux32.eb]: Updated command to reflect suggested changes and resubmitted

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node351.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/ef320f27eead4568cef1fa2fa92cd968 for a full test report.

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @stevecdavis!

**Pull request merged**